lib/google/gax/api_callable.rb

```diff
@@ -33,45 +33,54 @@
 
 module Google
   module Gax
-    # Converts an rpc call into an API call governed by the settings.
+    # Converts an RPC call into an API call.
     #
-    # In typical usage, +func+ will be a proc used to make an rpc request.
-    # This will mostly likely be a bound method from a request stub used to make
-    # an rpc call.
+    # In typical usage, +func+ will be a proc used to make an RPC request.
+    # This will mostly likely be a bound method from a request Stub used to make
+    # an RPC call.
     #
     # The result is created by applying a series of function decorators
-    # defined in this module to +func+.  +settings+ is used to determine
-    # which function decorators to apply.
+    # defined in this module to +func+.
     #
-    # The result is another proc which for most values of +settings+ has the
-    # same signature as the original.
+    # The result is another proc which has the same signature as the original.
     #
     # @param func [Proc] used to make a bare rpc call
-    # @param settings [CallSettings] provides the settings for this call
+    # @param timeout [Numeric] client-side timeout for API calls
```

Comment:
is this in seconds?

Reply:
Yes, it is seconds. I want to do a full documentation pass eventually.

Reply:
Ok, we can circle back then.